projectile.el

```diff
@@ -2387,6 +2381,12 @@ TEST-PREFIX which specifies test file prefix."
 (projectile-register-project-type 'nix '("default.nix")
                                   :compile "nix-build"
                                   :test "nix-build")
+(projectile-register-project-type 'rails-test '("Gemfile" "app" "lib" "db" "config" "test")
```

Comment:
I'd also add some comment here that the order matters and they should go after `npm`.

Reply:
OK.
I put a comment along with the link of this.